.drone.yml

```diff
@@ -30,7 +30,7 @@ pipeline:
     commands:
       - yarn clean
       - yarn dist
-      - 'yarn netlify deploy --site $${NETLIFY_SITE_ID} --message "Drone build: ${CI_BUILD_NUMBER}"'
+      - 'yarn netlify deploy --site $${NETLIFY_SITE_ID} --branch ${DRONE_COMMIT_BRANCH} --message "Drone build: ${CI_BUILD_NUMBER} for the branch: ${DRONE_COMMIT_BRANCH}"'
```

Comment:
Thanks for testing and making this!

There's a new possibility since yesterday to have `--alias` instead of `--branch` https://github.com/netlify/cli/issues/275#issuecomment-644473304

Optional update, I think `--alias` can be better

Let me know if you plan to update or not and i'll merge

Reply:
Yeah, why not, checked the multisite ticket and it has not been handled, yet ;)